File: day2Operator.cs
using System;

class solution
{
    static void Main(String[] args)
    {
        var mealCost = double.Parse(Console.ReadLine());
        var tipPercent = int.Parse(Console.ReadLine());
        var taxPercent = int.Parse(Console.ReadLine());
        
        var tip = tipPercent*mealCost/100;
        var tax = taxPercent*mealCost/100;
        
        var totalCost = Math.Round(tip + tax + mealCost);
        Console.WriteLine("The total meal cost is {0:#.#} dollars.", totalCost);
    }
}

File: day2Operator.cpp
#include <iostream>
#include <cmath>

using namespace std;
int main(){
    string tmp;
    
    getline(cin,tmp);
    double mealCost = stod(tmp);
    
    getline(cin,tmp);
    int tipPercent = stoi(tmp);
    
    getline(cin,tmp);
    int taxPercent = stoi(tmp);
    
    double tip = tipPercent*mealCost/100;
    double tax = taxPercent*mealCost/100;
    
    int totalCost = round(tip+tax+mealCost);
    
    cout<<"The total meal cost is "<<totalCost<<" dollars."<<endl;

}
